assets/sass/protocol/includes/themes/_firefox.scss

```diff
@@ -25,7 +25,7 @@
     --link-color-hover: #{tokens.$color-link-hover};
     --link-color-inverse: #{tokens.$color-blue-10};
     --link-color-visited-hover-inverse: #{tokens.$color-violet-05};
-    --link-color-visited-hover: #{tokens.$color-link-hover};
+    --link-color-visited-hover: #{tokens.$color-link-visited-hover};
```

Comment:
Another place that would be simplified by having the same hover state for both visited and unvisited links.